src/pip/_internal/utils/setuptools_build.py

```diff
@@ -20,17 +20,28 @@
 )
 
 
-def make_setuptools_shim_args(setup_py_path, unbuffered_output=False):
-    # type: (str, bool) -> List[str]
+def make_setuptools_shim_args(
+        setup_py_path,  # type: str
+        global_options=None,  # type: Sequence[str]
+        no_user_cfg=False,  # type: bool
+        unbuffered_output=False  # type: bool
+):
+    # type: (...) -> List[str]
     """
     Get setuptools command arguments with shim wrapped setup file invocation.
 
     :param setup_py_path: The path to setup.py to be wrapped.
+    :param global_options: Additional global options.
+    :param no_user_cfg: If True, disables personal user configuration.
     :param unbuffered_output: If True, adds the unbuffered switch to the
      argument list.
     """
     args = [sys.executable]
     if unbuffered_output:
         args.append('-u')
     args.extend(['-c', _SETUPTOOLS_SHIM.format(setup_py_path)])
+    if global_options:
+        args.extend(list(global_options))
```

Comment:
You don't need `list()` since `extend()` doesn't alter the argument passed to it.